analysis/src/analyzer.js

```diff
@@ -26,7 +26,25 @@ class UrlLoader extends FSUrlLoader {
    * Only return files from within the package directory.
    */
   async readDirectory(pathFromRoot, deep) {
-    return await new FSUrlLoader(this.packageRoot).readDirectory(pathFromRoot, deep);
+    const files = await new FSUrlLoader(this.packageRoot).readDirectory(pathFromRoot, deep);
+    const result = [];
+
+    // Filter out minified files etc.
+    for (const path of files) {
+      // eg. for file.min.js => file.js
+      const fileMatch = pathlib.basename(path).match(/([^.]*)(?:\.[^.]*)*(\.[^.]*)/);
```

Comment:
Kinda hard to read this regexp. Would probably just prefer using `split` or something. Up to you though.

Reply:
Thanks, changed to use `split`.